components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html

```diff
@@ -17,15 +17,15 @@
         <chef-heading>Profiles</chef-heading>
         <chef-subheading>Compliance profiles manage security and compliance scans.</chef-subheading>
 
-        <div class="profiles-search">
+        <div class="profiles-search" *ngIf="!profilesEmpty || isAvailableProfilesVisible">
           <input
             chefInput
             type="text"
             placeholder="Search profiles..."
             (input)="onSearchInput($event)"/>
         </div>
 
-        <chef-tab-selector class="profiles-tabs" [value]="selectedTab" (change)="onTabChange($event)">
+        <chef-tab-selector *ngIf="isAvailableProfilesVisible" class="profiles-tabs" [value]="selectedTab" (change)="onTabChange($event)">
```

Comment:
Only show the two tabs ![image](https://user-images.githubusercontent.com/1679247/83309079-30edc080-a1bd-11ea-9831-ef3859fe7381.png) if the available profiles are visible.